recipes/_post_install.rb

```diff
@@ -44,6 +44,7 @@
   PrivateTmp=true
   LimitNOFILE=1048576
   LimitNPROC=1048576
+  Restart=#{node['jlsolrcloud']['service_restart']}
```

Comment:
Do you have to do something like `systemctl reload` after this change / does Chef do that for you?

Reply:
I think chef-client does it for us because this is a `systemd_unit`.

I conducted some kitchen experiments to suggest it does indeed get applied after the chef-client run (I killed the solr process and it got restarted)